Pocket Intents/de.lproj/AppShortcuts.strings

```diff
@@ -0,0 +1,3 @@
+"Show me some cool stuff in ${applicationName}" = "Zeig mir coole Sachen in ${applicationName}";
```

Comment:
Should we setup this file path in smartling?

Reply:
eventually, though I am still trying to figure out if we can leverage our Localization package and SwiftGen, so the localization on Intents is still kinda WIP, but this works nonetheless.